pkgs/telemetry/src/LaunchDarkly.ServerSdk.Telemetry.csproj

```diff
@@ -37,7 +37,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LaunchDarkly.ServerSdk" Version="8.*" />
+    <PackageReference Include="LaunchDarkly.ServerSdk" Version="[8.7, 9.0)" />
```

Comment:
Hook 1.1.0 will require SDK 8.7.0 or greater. I constrained to less than 9 for now.